apps/crn-server/src/controllers/team.controller.ts

```diff
@@ -46,16 +46,7 @@ export default class TeamController {
 
     return {
       total,
-      items: items.map((team) => {
```

Comment:
For other reviewers that may come here, please look at this https://github.com/yldio/asap-hub/pull/3929#discussion_r1408059312

I've removed the `showTeamTools` only for `fetch` method, the `fetchById` remains the same, so the tools feature is still working

Reply:
So functionality may remain the same but are you sure we are not surfacing the tools for each time in the response now? As we are not meant to do that either